rollbar/__init__.py

```diff
@@ -702,6 +719,12 @@ def _report_exc_info(exc_info, request, extra_data, payload_data, level=None):
     if extra_trace_data and not extra_data:
         data['custom'] = extra_trace_data
 
+    if scopes:
+        data['scope'] = scopes[-1]
+
+        global scopes
```

Comment:
Why we want to clean the scopes here? It's possible sending more than one exception or message in one `watch()` block, so I think we want to let the context manager to remove the scopes on `__exit__` but not when sending data to Rollbar

Reply:
Ah. So, this was just one way that I thought of that could have mitigated the scope leak. The idea being to start a convention of always calling or having Rollbar report when a `watch()` block is used because essentially, whatever you wrap in a `watch()`, you'd want those errors to make it to Rollbar.

Reply:
mmm I am not sure I am getting the logic behind this yet, sorry. what is a "scope leak"? I think all reports within a `watch()` section should have the defined scopes, not only the first call to Rollbar, so I think we don't really want to remove the `scopes` here

Reply:
Ah. By "scope leak" I was referring to what Walt mentioned here https://github.com/rollbar/pyrollbar/pull/357#discussion_r519066313. I agree with you and I've changed it so that we don't clear scopes after we report to Rollbar.